frontend/src/v5/ui/controls/chip/chip.types.tsx

```diff
@@ -0,0 +1,183 @@
+		tooltip: formatMessage({ id: 'chip.priorityLevel.high.tooltip', defaultMessage: 'High priority' }),
+		color: COLOR.ERROR_MAIN,
+		icon: <FlagFillIcon />,
+	},
+};
+
+export const RISK_LEVELS_MAP = {
+	[RiskLevels.UNSET]: {
+		label: formatMessage({ id: 'chip.riskLevel.unset', defaultMessage: 'Unset Risk' }),
+		color: COLOR.BASE_LIGHT,
+	},
+	[RiskLevels.VERY_LOW]: {
+		label: formatMessage({ id: 'chip.riskLevel.veryLow', defaultMessage: 'Very Low Risk' }),
+		color: COLOR.FAVOURITE_MAIN,
+	},
+	[RiskLevels.LOW]: {
+		label: formatMessage({ id: 'chip.riskLevel.low', defaultMessage: 'Low Risk' }),
+		color: '#FF9800',
+	},
+	[RiskLevels.MODERATE]: {
+		label: formatMessage({ id: 'chip.riskLevel.moderate', defaultMessage: 'Moderate Risk' }),
+		color: '#ED6C02',
+	},
+	[RiskLevels.HIGH]: {
+		label: formatMessage({ id: 'chip.riskLevel.high', defaultMessage: 'High Risk' }),
+		color: COLOR.ERROR_MAIN,
+	},
+	[RiskLevels.VERY_HIGH]: {
+		label: formatMessage({ id: 'chip.riskLevel.veryHigh', defaultMessage: 'Very High Risk' }),
+		color: '#000000',
+	},
+};
+
+export const STATUS_MAP = {
+	[TicketStatuses.OPEN]: {
+		label: formatMessage({ id: 'chip.ticketStatus.open', defaultMessage: 'Open' }),
+		color: COLOR.BASE_MAIN,
+		icon: <StarIcon />,
+	},
+	[TicketStatuses.IN_PROGRESS]: {
+		label: formatMessage({ id: 'chip.ticketStatus.inProgress', defaultMessage: 'In Progress' }),
+		color: '#7156FF',
+		icon: <ClockIcon />,
+	},
+	[TicketStatuses.FOR_APPROVAL]: {
+		label: formatMessage({ id: 'chip.ticketStatus.forApproval', defaultMessage: 'For Approval' }),
+		color: '#0288D1',
+		icon: <BellIcon />,
+	},
+	[TicketStatuses.CLOSED]: {
+		label: formatMessage({ id: 'chip.ticketStatus.closed', defaultMessage: 'Closed' }),
+		color: '#2E7D32',
+		icon: <TickIcon />,
+	},
+	[TicketStatuses.VOID]: {
+		label: formatMessage({ id: 'chip.ticketStatus.void', defaultMessage: 'Void' }),
+		color: '#000',
+		icon: <CrossIcon />,
+	},
+};
+
+export const TREATMENT_LEVELS_MAP = {
+	[TreatmentLevels.UNTREATED]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.untreated', defaultMessage: 'Untreated' }),
+		color: COLOR.BASE_LIGHT,
+	},
+	[TreatmentLevels.PROPOSED]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.proposed', defaultMessage: 'Proposed' }),
+		color: '#0288D1',
+	},
+	[TreatmentLevels.AGREED_PARTIAL]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.agreedPartial', defaultMessage: 'Agreed (Partial)' }),
+		color: '#4CAF50',
+	},
+	[TreatmentLevels.AGREED_FULLY]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.agreedFully', defaultMessage: 'Agreed (Fully)' }),
+		color: '#2E7D32',
+	},
+	[TreatmentLevels.REJECTED]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.rejected', defaultMessage: 'Rejected' }),
+		color: COLOR.ERROR_MAIN,
+	},
+	[TreatmentLevels.VOID]: {
+		label: formatMessage({ id: 'chip.treatmentLevel.void', defaultMessage: 'Void' }),
+		color: '#000000',
+	},
+};
```

Comment:
It's a lot simpler to not have to create a new file every time we add a new chip and then reuse a lot of the same code. Especially since there are both Chip and ChipSelect. It's far easier (in my opinion) to have these two components act as templates.